docs/features/backup_and_restore.md

```diff
@@ -6,7 +6,201 @@ hide_title: false
+
+##### Disaster Recovery
+
+To recover a Cumulus Postgres database in a disaster or data-loss scenario, you should perform the following steps:
+
+* If the Postgres database cluster exists/is still online, halt workflow
+  activity, then take the cluster offline/remove access.
+* If needed, recover the DynamoDB tables as noted in the
+  [DynamoDb](./backup_and_restore#dynamodb) section of this document.
+* Redeploy a new database cluster from your backup, matching as closely as possible to the DynamoDB
+  restore time.   See [AWS's
+  PIT recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html)
+  and [DB Snapshot recovery
+  instructions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html),
+  or the examples below for more information.
+* Configure your Cumulus deployment to utilize the new database cluster and re-deploy.
+* Run DynamoDB/RDS reconciliation tools and resolve any discrepancies.
+
+##### cumulus-rds-tf examples
+
+The following sections provide a walk through of a few recovery scenarios for the provided `cumulus-rds-tf`
+serverless module.
+
+***Point In Time Recovery***
+
+If you need recovery that exceeds the 1-day granularity of AWS's snapshots, you
+either must create and manually manage snapshots, or use Point In Time
+Recovery (PITR) if you still have the original cluster available.
+
+Unfortunately as terraform does not yet support RDS PITR (see:
+[github terraform-provider issue #5286](https://github.com/terraform-providers/terraform-provider-aws/issues/5286)),
+this requires a manual procedure.
+
+If you are using the `cumulus-rds-tf` module to deploy an RDS Aurora Serverless
+Postgres instance, the following procedure can be used to successfully spin up a duplicate
+cluster from backup in recovery scenarios where the database cluster is still viable:
+
+#### **1. Halt all ingest/remove access to the database to prevent Core processes from writing to the old cluster.**
+
+##### Halt Ingest
+
+Deactivate all Cumulus Rules, halt all clients that access the archive API and
+stop any other database accessor processes.   Ensure all active executions have
+completed before proceeding.
+
+##### Remove Database Cluster Access
+
+Depending on your database cluster/setup, there are several ways to limit access to the
+database.   One example:
+
+Log in as the administrative user to your database cluster and run:
+
+```sql
+alter database my_database connection limit 0;
+select pg_terminate_backend(pg_stat_activity.pid) from pg_stat_activity where pg_stat_activity.datname = 'database';
+```
+
+This should block new connections to the Core database from the database user
+and cause database writes to fail.
+
+Take care that you aren't using the admin user (e.g. postgres) with a default
+database or you may inadvertently limit connections by your administrative
```

Comment:
Is it the admin user or the default database that is the issue here? What 'administrative tasks'? Postgres-internal tasks?

Reply:
If you reduce connection counts *and* kill existing connections to your admin user inadvertently you could make it impossible to connect as the admin user, or access the database <things> are using to store config/values/?

Reply:
Probably this could be vaugeified with a warning that says "make sure you're using the correct db user/database" ?

Reply:
Really though, you don't want to be running with a situation where you have *one* database defined (e.g. postgres) and you limit your access to it via your administrative user.